distribution/resources/conf/default-micro-gw.conf.template

```diff
@@ -165,6 +165,8 @@
 
 # Analytics configurations
 [analytics]
+  # The configured API Manager analytics version
+  version = "3.2.0"
```

Comment:
Lets use this as stream version, because in APIm analytics 3.3.0 also might have the 3.2.0 streams if there are no changes

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/1319/commits/4c690770f6f86c3a209ac3bb9cc5d76a85640091